native/src/main/java/io/ballerina/stdlib/graphql/runtime/schema/SchemaRecordGenerator.java

```diff
@@ -154,6 +158,14 @@ private BArray getFieldsArray(SchemaType schemaType) {
         return fieldsArray;
     }
 
+    private BArray getInputFieldsArray(SchemaType schemaType) {
```

Comment:
There's already a `getInputValueArray` method. Why do we need another function?

Reply:
since we are accessing `inputFields` , we can't use `getInputValueArray` here.